databases/week2/diagram.sql

```diff
@@ -0,0 +1,51 @@
+CREATE DATABASE diagram_db
+USE diagram_db
+-- -----------------------------------------------------
+-- Table`Products`
+-- -----------------------------------------------------
+CREATE TABLE Products (
+  `product_id` INT UNSIGNED NOT NULL AUTO_INCREMENT,
+  `item is stock` VARCHAR(45) NOT NULL,
+  `description` MEDIUMTEXT NULL,
+  `Status` VARCHAR(45) NOT NULL,
+  PRIMARY KEY (`product_id`);
+
+
+-- -----------------------------------------------------
+-- Table `Orders`
+-- -----------------------------------------------------
+CREATE TABLE Orders (
+  `Orders` INT NOT NULL AUTO_INCREMENT,
+  `item` VARCHAR(45) NULL,
+  `quantity` INT NOT NULL,
+  `color` VARCHAR(45) NULL,
```

Comment:
this is belong the to product table. A product has only one color in the system. A red car is separate item than a blue car.